packages/l10n/scripts/generate-l10n-data.js

```diff
@@ -294,6 +296,114 @@ const mapDiffToWarnings = (oldData, newData) => {
     .filter(Boolean);
 };
 
+const updateTimeZoneData = async (key) => {
+  const allTimeZoneIds = moment.tz.names();
+  // uncomment for testing/review purposes, will be removed once PR is out of draft
+  // const fakeTimeZones = ['zzspace/moon', 'zzspace/mars', 'zzspace/jupiter'];
+  // const allTimeZoneIds = moment.tz.names().concat(fakeTimeZones);
+  const dataFolderPath = path.join(__dirname, '..', DATA_DIR[key].path);
+  const sourceDataFilePath = path.join(dataFolderPath, 'core.json');
+  await fs.accessSync(sourceDataFilePath, fs.constants.F_OK);
+  const translatedTimeZones = JSON.parse(
```

Comment:
Coming in as a reviewer I struggle with naming and flow a bit. Just as a note. Will read the code multiple times to suggest something to improve it.

Reply:
fwiw, I freely admit that naming and flow could be improved, I'll think about it as well.